.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +138,34 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+results_file = f"gs://{args.artifacts_bucket}/{args.build_id}"
+if not results_file.endswith(".csv"):
+    results_file = results_file + ".csv"
+
 if args.test_percent == 100:
     notebooks = changed_notebooks
 else:
-    notebooks = [changed_notebook for changed_notebook in changed_notebooks if random.randint(1, 100) < args.test_percent]
+    notebook_results = execute_changed_notebooks_helper.load_results(results_file)
+
+    notebooks = [changed_notebook for changed_notebook in changed_notebooks if execute_changed_notebooks_helper.select_notebook(changed_notebook, notebook_results) < args.test_percent]
```

Comment:
Can you pass `args.test_percent ` into `execute_changed_notebooks_helper.select_notebook` so it's more testable?

Reply:
done